### PR TITLE
stable/sematext-agent: Update autodisco config

### PR DESCRIPTION
**Pull request by rabbitstack**
#### What this PR does / why we need it:

Update the autodisco config due to format changes.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x ] Chart Version bumped
- [x ] Variables are documented in the README.md
- [x ] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/sematext-agent/files/autodisco.yml [hunk 1]
```diff
@@ -37,242 +28,167 @@
...
-    - 8093
-    - 8094
-    - 11210
+  name: Couchbase
+  signatures:
+    - beam.smp couchbase
   labels:
-    COUCHBASE_HOSTPORT: "{{ .IP }}:8093"
+    COUCHBASE_HOST_PORT: "{{ .IP }}:8093"
     COUCHBASE_QUERY_HOSTPORT: "{{ .IP }}:8093"
```
Comment (hakman) [inline stable/sematext-agent/files/autodisco.yml]: This remains `HOSTPORT`?

File: stable/sematext-agent/files/autodisco.yml [hunk 2]
```diff
@@ -37,242 +28,167 @@
...
-    - 9300
-    - 9201
-    - 9202
-    - 9301
-    - 9302
+  name: Elasticsearch distributed search engine
+  signatures:
+    - java org.elasticsearch.bootstrap.Elasticsearch
   labels:
     SPM_MONITOR_ES_NODE_HOSTPORT: "{{ .IP }}:9200"
```
Comment (hakman) [inline stable/sematext-agent/files/autodisco.yml:67]: This remains `HOSTPORT`?

File: stable/sematext-agent/files/autodisco.yml (hunk 1 above)
Comment (sivasamyk) [reply stable/sematext-agent/files/autodisco.yml]: Will fix.

File: stable/sematext-agent/files/autodisco.yml (hunk 2 above)
Comment (sivasamyk) [reply stable/sematext-agent/files/autodisco.yml:67]: Yes. We decided to leave the conf variables for old integrations as-is

**Pull request merged**